jaeseon/app.js

```diff
@@ -49,6 +48,48 @@ app.post('/users', async (req, res) => {
   return res.status(201).json({ message: 'userCreated' });
 });
 
+app.post('/posts', async (req, res) => {
+  const { title, content, user_id } = req.body;
+  await myDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      user_id
+    ) VALUES (?, ?, ?);
+    `,
+    [title, content, user_id]
+  );
```

Comment:
```suggestion
  await myDataSource.query(
    `INSERT INTO posts(
      title,
      content,
      user_id
    ) VALUES (?, ?, ?);
    `,
    [title, content, userId]
  );
```
DB에는 posts 테이블에 user_id라고 정의되어 있기에 query method에 들어가는 SQL 문 자체는 `user_id`로 들어가되 ?에 들어갈 값으로는 위의 코멘트에 따라 수정되어야겠습니다.